internal/controllers/cryostat_controller_test.go

```diff
@@ -1418,6 +1429,17 @@ var _ = Describe("CryostatController", func() {
 				Expect(kerrors.IsNotFound(err)).To(BeTrue())
 			})
 		})
+		Context("Cryostat CR has authorization properties", func() {
+			BeforeEach(func() {
+				t.objs = append(t.objs, test.NewCryostatWithAuthProperties(), test.NewAuthPropertiesConfigMap())
+			})
+			JustBeforeEach(func() {
+				t.reconcileCryostatFully()
+			})
+			It("Should add volumes and volumeMounts to deployment", func() {
+				t.checkDeploymentHasAuthProperties()
+			})
+		})
```

Comment:
This test could instead verify that specifying the authorization properties when running on Kubernetes does nothing.